src/SFA.DAS.EAS.Employer_Financial.Database/Scripts/PreDeployment/AddLevyDeclarationCreatedDateDefault.sql

```diff
@@ -0,0 +1,22 @@
+﻿-- This script was created as there was already a fixup script that had not fully worked in Production
```

Comment:
At first glance I can't tell what this script is doing. If it's adding a default value to a column can this be added to the table's definition instead of a predeployment script?